wlutil/br/br.py

```diff
@@ -82,22 +64,19 @@ def buildConfig():
     # Our config fragment, specifies differences from the default
     marshalKfrag = br_dir / 'buildroot-config'
 
-    # We're just borrowing linux's merge_config.sh helper since br uses the same make system
-    # This doesn't actually depend on any details of this linux kernel
-    mergeScript = str(pathlib.Path(wlutil.linux_dir) / 'scripts' / 'kconfig' / 'merge_config.sh')
+    mergeScript = br_dir / 'merge_config.sh'
```

Comment:
I had to copy this out of linux because the the buildroot image might be built before the binary (where the linux submodule check happens). This is also probably a bit cleaner anyway.